### ISSUE TITLE
[Suggestion] Roleplaying & Worldbuilding Tools.

### ISSUE THREAD

**Issue by GeneralScoopz** [tsunamayo/Starship-EVO#2254]
Hi hi,

Not sure how much this has been thought about outside missions, but I thought I'd bring it up after thinking about it forever.

In short, I think it would add a ton of value to open up the construction of the world (so to speak) to the players. The one thing that seems constant across all the games in the genre are roleplaying servers. Possibly because in the past factions and other deeper gameplay systems have always come late or been lacking, but regardless, I think that there will always be a sizable chunk of people who will desire to start creating experiences for their fellow players. To that end, I think most people would value the following:

---
-- Some sort of crew system. How much they actually DO is up for debate, but an empty ship is definitely a bummer.

-- player-made NPCs, placing merchants, vendors, or questgivers is a great way to populate player stations and probably crucial in keeping people engaged with the world of STEVO.

-- Custom quests/dialogue. Again, how complex a system like that would be I cannot say, but what a great way to establish lore, faction identities, and just **reasons to care** about a ship or crew or place.

-- A cutscene editor? This one is probably a pretty tall order to be honest, but if sometime in the future there was time to flesh things out more, it would be a great way to add production value and engagement to custom content.

-- The human details. Comms chatter, AI ships actually docking, foot traffic inside vessels. The small stuff goes a long way when we talk about worldbuilding, whether its just movement or seeing an NPC stop a captain on a customs check or whatever.

---
I could probably say a lot more and really get into the minutiae of it all, but I feel like that last thing to do is to make this seem overwhelming (and that list is by no means exhaustive). I just think it's important to have CONTEXT in the game, to have reasons for doing things other than "Shiny coin mean bigger, faster shooty tube!" There's nothing worse than an empty universe and I think the games that shy away from answering the tough questions about A.) what their universe is like, and B.) how the players interact with it on a human level are the ones that slowly die out.

So yeah, that's my two cents, having been a backer from the start I already love the game! But I think these kind of tools and opportunities would elevate STEVO from good game to legendary status.

**Comment by tsunamayo** [comment]
Yes I do agree with all of that. There is two things here, how much lore we should have in the base vanilla game, and making tool for the player to mod it.
All I can say for now is that I am working on making faction and stuff in a easily moddable way (you can change file in the install folder). I am gonna have a serious look at the workshop also of course. Thanks

**Comment by ZachZent** [comment]
I will add that people will no matter how much it is supported, players will have some sort of faction vs faction gameplay. When this part of the game is added, probably during or after dedicated servers are added, please add many faction control features. Simple things from team chat to advanced features such as the ability to add player ranks (which ranks can fly/spawn which ships, access faction supplies and money, etc.) The more user control that is added in the base game, the more players will have fun using them. Factions can extend to random people recruited online looking to join a large faction in a server without fear that some troll will come along and steal all the money. As for the actual features that could be added will probably be talked about in some post far down the line, but it is a good thing to keep in mind.

**Comment by tgamorris76** [comment]
Faction play would be very welcome, planning on a Battletech mod whne the mechs come out
